core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -698,6 +700,9 @@ DEFAULT_KAFKASTORE_WRITE_MAX_RETRIES, atLeast(0),
     .define(SCHEMAREGISTRY_RESOURCE_EXTENSION_CONFIG, ConfigDef.Type.LIST, "",
             ConfigDef.Importance.LOW, SCHEMAREGISTRY_RESOURCE_EXTENSION_DOC
     )
+    .define(ENABLE_FIPS_CONFIG, ConfigDef.Type.BOOLEAN, "false",
+        ConfigDef.Importance.LOW, ""
+    )
```

Comment:
i don't think we need this given that it's already defined in security-plugins

Reply:
I have no way of testing this currently, but I think the CI build might fail if I don't include it